src/main/java/edu/ucsb/cs156/happiercows/controllers/CommonsController.java

```diff
@@ -110,7 +110,7 @@ public ResponseEntity<String> updateCommons(
     updated.setMilkPrice(params.getMilkPrice());
     updated.setStartingBalance(params.getStartingBalance());
     updated.setStartingDate(params.getStartingDate());
-    updated.setEndingDate(params.getEndingDate());
+    // updated.setEndingDate(params.getEndingDate());
```

Comment:
same..  most organizations try to avoid committing commented out code to `main`

Reply:
got it, good to know